### PR TITLE
Create Ent-trimmer plugin

### PR DESCRIPTION
**Pull request by pwatts6060**

### CODE REVIEW

**Comment by aHooder** [comment]
Closing & re-opening to re-run the checks.

**Review by aHooder — CHANGES_REQUESTED** [review]
You need to synchronize the `ImageIO.write` call at https://github.com/pwatts6060/runelite-plugins/blob/81a327c542c807336993c291730154fd88f22dc5/src/main/java/com/ent/discord/DiscordWebhook.java#L113 to appease the checker, like so:
```java
synchronized (ImageIO.class)
{
	ImageIO.write(bufferedImage, "png", byteArrayOutputStream);
}
```
LGTM otherwise

**Review by aHooder — APPROVED** [review]

**Comment by aHooder** [comment]
Your PR will have to wait for a maintainer to review it, due to the HTTP request, but this looks alright to me now. For some reason, it's still warning about ImageIO synchronization though :thinking: Presumably it's unable to check for the synchronization block.

**Review by Nightfirecat — APPROVED** [review]

**Pull request merged**
